docs/UserGuide.md

```diff
@@ -186,16 +185,6 @@ Example:
 * `findstu john mary`
 
 
-### Listing all students: `liststu`
-
-Lists all students in the student list.
-
-Format: `liststu`
-
-Example: 
-
-* `liststu`
-
```

Comment:
Might need to trouble you to add the details back for this part too.

Reply:
Added back in.